entrypoint.sh

```diff
@@ -8,6 +8,11 @@ skip_if_exists=$4
 variables=$5
 filename=$6
 
+if [ -z $name ]; then
```

Comment:
@jLopezbarb we should mark this mas mandatory on the github action yaml as well

Reply:
It was already marked but it seems that its just informative. Now users can have better error understanding